api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/dsd/NextAndPrevDatesCalculation.java

```diff
@@ -96,6 +102,7 @@ public int compare(Obs obs, Obs t1) {
           scheduled = true;
         }
       }
+      scheduled = true;
```

Comment:
Why add this here?

Reply:
I have cleaned this up as well. @nthfloor